Day5/groupApp/src/Navbar/Navbar.js

```diff
@@ -0,0 +1,112 @@
+import img from "../../images/reactLogo.png";
+import imgA from "../../images/avatar1.png";
+import searchIcon from "../../images/search.png";
+import './style.css';
+import { useState } from "react";
+const data= [
```

Comment:
Make a single JSON file and import data from there to both App.js and Navbar.js

Reply:
Thank you for your feedback, will made changes accordingly